docker-compose.yaml

```diff
@@ -109,7 +109,7 @@ services:
   demo:
     image: ghcr.io/saveourtool/save-demo:${DEMO_TAG}
     environment:
-      - "SPRING_PROFILES_ACTIVE=${PROFILE}"
+      - "SPRING_PROFILES_ACTIVE=${PROFILE},docker-secrets"
```

Comment:
I guess you don't use authorization & authentication in save-demo yeat?

Reply:
Yes, for now I do not use it as far as all endpoints either public or internal. Maybe later there will be endpoints that require authorization